addons/website/views/website_templates.xml

```diff
@@ -1946,40 +1949,44 @@
     </a>
 </template>
 
-<template id="language_selector" inherit_id="portal.language_selector">
-    <xpath expr="//t[@t-nocache]" position="attributes">
-        <attribute name="t-nocache-flags">flags</attribute>
-    </xpath>
+<template id="language_selector" inherit_id="" name="Language Selector">
```

Comment:
```suggestion
<template id="language_selector" name="Language Selector">
```
Code leftover?

Reply:
Never saw that, it will be the first one in the code base, I guess it works but it's really strange and unusual.
Why not just write a migration script as it is supposed to be?
But actually, it should be working migration wise without it, it's not in a `noupdate` section: the XML view should get the update and it should also be replicated on COW views (if not, it's a bug to fix, let me know).